tox.ini

```diff
@@ -13,36 +13,32 @@ extras =
     test
 
 commands =
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} bootstrap
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} annotate
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} install test
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir}
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src bootstrap
```

Comment:
yes that makes sense, due to buildout internals to place sources-dir relative to buildout:directory